### PR TITLE
Fix some problems and version up

### PR DESCRIPTION
**Pull request by johngrib**
I fixed some problems. And I use this.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,47 +1,51 @@
...
-        classpath(kotlin("gradle-plugin", "1.3.21"))
-    }
+plugins {
+    id("org.jetbrains.intellij") version "1.13.3"
+    kotlin("jvm") version "1.8.20"
 }
 
-plugins {
-    id("org.jetbrains.intellij") version "0.4.7"
-    kotlin("jvm") version "1.3.21"
```
Comment (johngrib) [inline build.gradle.kts:15]: org.jetbrains : 0.4.7 -> 1.13.3

File: src/main/kotlin/org/beeender/neovim/Client.kt [hunk 2]
```diff
@@ -55,7 +55,7 @@ class Client(connection: NeovimConnection, onClose: (Throwable?) -> Unit) {
 
     init {
         receiver.start({msg ->
-            ReceiverChannel.offer(this to msg)
+            ReceiverChannel.trySend(this to msg)
```
Comment (johngrib) [inline src/main/kotlin/org/beeender/neovim/Client.kt:58]: `offer` is deprecated.

**Comment by vegerot** [comment]
@johngrib I think this project is dead.  Do you want to fork this plugin?

**Comment by beeender** [comment]
@johngrib I invited you as collaborator to this project if you are interested.

Sorry I missed this PR before.

**Review by beeender — APPROVED** [review]
Thanks a lot for the PR! (Travis CI is gone, I probably need to move it to the GitHub actions later.)

**Pull request merged**